app/MainHie.hs

```diff
@@ -120,15 +126,15 @@ run opts = do
   d <- getCurrentDirectory
   logm $ "Current directory:" ++ d
 
-  let vomitOptions = defaultOptions { boLogging = BlVomit}
-  let defaultOpts = if optGhcModVomit opts then vomitOptions else defaultOptions
+  let initOpts = defaultCradleOpts { cradleOptsVerbosity = verbosity }
```

Comment:
Verbosity is not yet passed on to `hie-bios`.

Reply:
Yes I need to replace all the calls to `pprTrace` in `hie-bios` with explicit logging.